flink-table/flink-table-planner/src/test/java/org/apache/flink/connector/file/table/FileSystemStatisticsReportTest.java

```diff
@@ -190,17 +189,23 @@ public void testFilterPushDownAndReportStatisticsDisabled() {
     }
 
     @Test
-    public void testNoPartitionPushDownAndCatalogStatisticsExist()
-            throws PartitionNotExistException {
+    public void testLimitPushDownAndCatalogStatisticsDoNotExist() {
```

Comment:
add a test about limit push down and statistics is unknown

Reply:
> add a test about limit push down and statistics is unknown

done!